File: javascript-closure-angular-client/API/Client/Weapon.js
goog.provide('API.Client.Weapon');

/**
 * @record
 */
API.Client.Weapon = function() {}

/**
 * - '0' # restrict use - '1' # public use 
 * @type {!string}
 * @export
 */
API.Client.Weapon.prototype.weaponRestrictionType;

/**
 * @type {!string}
 * @export
 */
API.Client.Weapon.prototype.serieNumber;

/**
 * Barrel's serial number. Número de série do cano 
 * @type {!string}
 * @export
 */
API.Client.Weapon.prototype.gunBarrelSerieNumber;

/**
 * weapon description
 * @type {!string}
 * @export
 */
API.Client.Weapon.prototype.weaponDescription;

/** @enum {string} */
API.Client.Weapon.WeaponRestrictionTypeEnum = { 
  0: '0',
  1: '1',
}

File: async-scala-client/src/main/scala/io/swagger/client/model/Weapon.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Weapon (
  weaponRestrictionType: String,  // - &#39;0&#39; # restrict use - &#39;1&#39; # public use 
serieNumber: String,
gunBarrelSerieNumber: String,  // Barrel&#39;s serial number. Número de série do cano 
weaponDescription: String  // weapon description
)
